bedrock/firefox/templates/firefox/welcome/page1.html

```diff
@@ -18,11 +18,19 @@
 
 {% set _entrypoint = 'mozilla.org-firefox-welcome-1' %}
 {% set _utm_campaign = 'welcome-1-monitor' %}
+{% set _utm_content = 'stay-ahead-hackers' %}
```

Comment:
I think we should also put this value behind an l10n tag so that the analytics only change when the string is available?

Reply:
Good idea, though I expect this will be available today anyway. But it couldn't hurt.